examples/vae/README.md

```diff
@@ -12,6 +12,10 @@ python main.py
 
```

Comment:
Git doesn't let me comment on parts that are not considered different. But could you please remove a semicolon after `## Examples` ?